src/netex_idf/offers.rs

```diff
@@ -432,19 +447,103 @@ where
         .collect()
 }
 
-fn parse_passenger_stop_assignment<'a, I>(psa_elements: I) -> HashMap<String, String>
+pub fn get_or_create_stop_point(
```

Comment:
I just realized, maybe the name of this function is not ideal because neither the `StopPoint` nor the `VirtualStopPoint` is created. Would be more something like `get_real_or_virtual_stop_point`... or maybe just `get_stop_point`?

Reply:
You push a virtual stoppoint in the collection.stoppoints. `get_stop_point` if you prefer.

Reply:
`get_stop_point` is fine for me.

Reply:
Ok. I will do that now. 
More tomorrow. I am dead :laughing: